chromium/browser_type.go

```diff
@@ -312,13 +312,21 @@ func prepareFlags(lopts *common.LaunchOptions, k6opts *k6lib.Options) map[string
 		f["mute-audio"] = true
 		f["blink-settings"] = "primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4"
 	}
+	ignoreDefaultFlags(f, lopts.IgnoreDefaultArgs)
 
 	setFlagsFromArgs(f, lopts.Args)
 	setFlagsFromK6Options(f, k6opts)
 
 	return f
 }
 
+// ignoreDefaultFlags ignores any flags in teh provide slice
+func ignoreDefaultFlags(flags map[string]interface{}, toIgnore []string) {
```

Comment:
Minor, WDYT of the following?

```suggestion
func ignoreDefaultArgsFlags(flags map[string]interface{}, toIgnore []string) {
```